setup.py

```diff
@@ -2,7 +2,7 @@
 import os, platform, sys
 
 # look for environment variable that specifies path to SCIP Opt lib and headers
-scipoptdir = os.environ.get('SCIPOPTDIR', '')
+scipoptdir = os.environ.get('SCIPOPTDIR', '/usr/local')  # assume SCIP shared library is in /usr/local if SCIOPTDIR not set
```

Comment:
This is only correct/useful for Linux and Mac, so we need to distinguish the platform.

Reply:
@darrenrhea, I will merge your changes as soon as you fix this one issue.